lib/mjml/parser.rb

```diff
@@ -64,11 +74,28 @@ def validation_level
       "--level=#{MJML.config.validation_level}" if MJML::Feature.available?(:validation_level)
     end
 
+    def output_from_file
+      @temp_file = Tempfile.new("mjml-template")
+      _out, err, _sts = Open3.capture3(cmd, stdin_data: @template)
+      @temp_file.rewind
+      @temp_file.unlink
+      return @temp_file.read, err
+    end
+
+    def output_from_memory
+      out, err, _sts = Open3.capture3(cmd, stdin_data: @template)
+      return out, err
+    end
+
+    def should_get_outpout_from_file?
+      @template.size > 20_000
```

Comment:
I didn't know stdin has some limitations. Could you send some link to read about this number or it is random?